EIPS/eip-3770.md

```diff
@@ -36,7 +36,7 @@ Chain-specific address = "`shortName`" "`:`" "`address`"
 `address` is mandatory and MUST be a EIP-55 compatible hexadecimal address
 
 ### Examples
-![Chain-specific addresses](../assets/eip-3770/EIP-3770-examples.png "Examples of chain-specific addresses")
+![Chain-specific addresses](../assets/eip-3770/examples.png "Examples of chain-specific addresses")
```

Comment:
I still don't get why this has to be a PNG and not just text.

Reply:
I argue that this shouldn't be included in the EIP at all since it is a "list of things" and EIPs are not appropriate for tracking "lists of things".  There is discussion about creating some kind of Ethereum registry system, but that would be outside of the scope of the EIP process.